### PR TITLE
Update and rework federated sharing page

### PR DESCRIPTION
**Pull request by voroyam**
Included the info from https://github.com/owncloud/docs/issues/133

Also clarified the process to establish federation.

Removed the oC 8 references.

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc [hunk 1]
```diff
@@ -3,150 +3,63 @@
...
 
-Federated Cloud Sharing is now managed by the Federation app (9.0+), and
-is now called Federation sharing. When you enable the Federation app you
-can easily and securely link file shares between ownCloud servers, in
-effect creating a "cloud" of ownCloud installations.
+Federated Cloud Sharing is managed by the Federation app. When you enable the Federation app you can easily and securely link file shares between ownCloud servers, in effect creating a "cloud" of ownCloud installations.
 
 [IMPORTANT]
 ====
 For security reasons federated sharing **strictly requires HTTPS (SSL/TLS)**.
```
Comment (pako81) [inline modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: It would actually work even with HTTP when the config.php option `'sharing.federation.allowHttpFallback'` is set to `true`. Not sure it is worth mentioning this, but this option is officially documented in our docs.

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc [hunk 2]
```diff
@@ -3,150 +3,63 @@
...
-remote ownCloud server. In this example, that is
-`freda@https://example.com/owncloud`. When ownCloud verifies the link,
-it displays it with the *(remote)* label. Click on this label to establish the link.
-+
-image:configuration/files/federation-2.png[image]
-5. When the link is successfully completed, you have a single share
-option, and that is *can edit*.
+. Verify valid SSL certificates on both servers.
+. Verify your `overwrite.cli.url` is configured to the correct URL and instead of `localhost`.
+. Reset your `oc_jobs` table.
```
Comment (pako81) [inline modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Why is this needed?

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc [hunk 3]
```diff
@@ -3,150 +3,63 @@
...
+occ dav:sync-system-addressbook
+occ federation:sync-addressbook
 ----
-. The `federatedfilesharing` app's `auto_accept_trusted config` option is set to `yes`; for example:
+. Configure automatic acceptance of new federated shares.
 +
-[source,php]
 ----
+occ config:app:set federation auto_accept_trusted --value '0'
 occ config:app:set federatedfilesharing auto_accept_trusted --value 'yes'
```
Comment (pako81) [inline modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Those options could also be set/unset in the admin page. Worth mentioning it.

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Have you tested this?

I remember me trying to set it up without certificates, in http mode, and I failed. Did you manage to get it to work?

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 2 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Because it's clean. We don't know if the federation cron job already run or not. Setting everything to 0 at the start creates good conditions for the federation to work.

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 3 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Yes, but I found a warning message in the docs, that if you check this option in the webUI it **won't** work. So to be on the save side - better set it with occ.

**Comment by voroyam** [comment]
Thanks for reviewing it @pako81

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Also maybe it's better to have in the docs the "save" method, so people don't come to us complaining the have a security flaw just because they followed the docs :)

Comment (pako81) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Yes, i tested this option successfully.

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 2 above)
Comment (pako81) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: Do this only for the federation background job not for all jobs.

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 3 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: > Yes, but I found a warning message in the docs, that if you check this option in the webUI it **won't** work. So to be on the save side - better set it with occ.

Makes sense!

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 2 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: @pako81, which class column value do you filter on to only clean out federation background jobs?

Comment (pako81) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: class is `OCA\Federation\SyncJob`

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: added

File: modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc (hunk 2 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc]: filtered

**Comment by voroyam** [comment]
works for me. @pako81 can you review this final state and then if you are happy with the changes we can merge it.

**Comment by pako81** [comment]
LGTM thanks
